Makefile

```diff
@@ -52,37 +51,75 @@ test-unit-debug:	build lair
 	LOG_LEVEL=silly NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/unit/
 	make stop-lair
 lair:
-	mkdir -p ./tests/tmp
-	rm -rf ./tests/tmp/*
-	mkdir -p ./tests/tmp/shim
-	RUST_LOG=trace lair-keystore --lair-dir tests/tmp/keystore &> hc-lair.log &
+	# Not clearning because on setup we generate keys for the host agents, deleting this file deletes all the keys
+	#rm -rf ./script/install-bundles/keystore
```

Comment:
Why would deleting this keystore folder cause a problem? Do the tests hard code a specific public key for the host?

Reply:
The keystore cannot be cleaned up between setup and running the test. 
That is why I move the clean up before the setup.